data/blog/opentelemetry-docker.mdx

```diff
@@ -192,15 +192,46 @@ receivers:
         enabled: true
       container.network.io.usage.tx_packets:
         enabled: true
+    hostmetrics:
+    collection_interval: 60s
+    scrapers:
+      cpu: {}
+      disk: {}
+      load: {}
+      filesystem: {}
+      memory: {}
+      network: {}
+      paging: {}
+      process:
+        mute_process_name_error: true
+        mute_process_exe_error: true
+        mute_process_io_error: true
+      processes: {}
+  prometheus:
+    config:
+      global:
+        scrape_interval: 60s
+      scrape_configs:
+        - job_name: otel-collector-binary
+          static_configs:
+            - targets:
+              # - localhost:8888
```

Comment:
This is an invalid yaml and this is not needed for this blog because we are only interested in the `docker_stats` receiver. Users can make additional changes but that's not our concern.